README.rst

```diff
@@ -1083,7 +1179,7 @@ If you want to extend this view, you will have to perform the additional steps b
         # add your customizations here ...
         pass
 
-**Step 2: remove the following line from your root ``urls.py`` file:**
+Step 2: remove the following line from your root ``urls.py`` file:
```

Comment:
**Note**: This is a very small refactoring in `Readme`, not related to this PR. Same for below change.